website/content/docs/what-is-vault.mdx

```diff
@@ -51,16 +65,75 @@ The key features of Vault are:
   Revocation assists in key rolling as well as locking down systems in the
   case of an intrusion.
 
-## What is HCP Vault?
+-> **Tip:** Learn more about [Vault use cases](/docs/use-cases).
+
+### What is HCP Vault?
+
+HashiCorp Cloud Platform (HCP) Vault is a hosted version of Vault, which is operated by HashiCorp to allow organizations to get up and running quickly. HCP Vault uses the same binary as self-hosted Vault, which means you will have a consistent user experience. You can use the same Vault clients to communicate with HCP Vault as you use to communicate with a self-hosted Vault.
+
+-> **Tip:** Learn more about [HCP Vault](https://cloud.hashicorp.com/docs/vault) or [get started](https://learn.hashicorp.com/collections/vault/cloud) by setting up a managed Vault cluster.
+
+### Why HCP Vault?
```

Comment:
Unless this is a requirement from someone above our pay grade, I would consider this trimming this down to the first paragraph and linking out to the HCP site. It might be more information than the visitor might want at this stage if they're just trying to learn about Vault.

Reply:
I might be able to convince the powers  to relocate some of the content to the actual HCP Vault docs :)